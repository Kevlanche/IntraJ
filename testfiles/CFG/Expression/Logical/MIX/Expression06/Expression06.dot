digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[26]"; "Opt[27]"; "Exit[28]"}
{ rank = same; "List[15]"; "ParameterDeclaration[18]"; "ParameterDeclaration[22]"; "Block[28]"}
{ rank = same; "Modifiers[19]"; "PrimitiveTypeAccess[21]"; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "List[29]"}
{ rank = same; "List[20]"; "List[24]"; "VarDeclStmt[30]"}
{ rank = same; "Modifiers[31]"; "ArrayTypeAccess[33]"; "List[36]"}
{ rank = same; "List[32]"; "ArrayTypeAccess[34]"; "VariableDeclarator[37]"}
{ rank = same; "PrimitiveTypeAccess[35]"; "List[38]"; "Opt[39]"}
{ rank = same; "ArrayInit[40]"}
{ rank = same; "List[41]"}
{ rank = same; "ArrayInit[42]"; "ArrayInit[58]"}
{ rank = same; "List[43]"; "List[59]"}
{ rank = same; "AndLogicalExpr[44]"; "OrLogicalExpr[51]"; "AndLogicalExpr[60]"}
{ rank = same; "GTExpr[45]"; "LTExpr[48]"; "GTExpr[52]"; "LTExpr[55]"; "GTExpr[61]"; "LTExpr[64]"}
{ rank = same; "VarAccess[46]"; "IntegerLiteral[47]"; "VarAccess[49]"; "IntegerLiteral[50]"; "VarAccess[53]"; "IntegerLiteral[54]"; "VarAccess[56]"; "IntegerLiteral[57]"; "VarAccess[62]"; "IntegerLiteral[63]"; "VarAccess[65]"; "IntegerLiteral[66]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Expression06 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[18]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[19]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[21]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[22]" [label="ParameterDeclaration\n < c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[27]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[28]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[29]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[30]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[31]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayTypeAccess[33]" [label="ArrayTypeAccess\n < boolean[] >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayTypeAccess[34]" [label="ArrayTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[35]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[37]" [label="VariableDeclarator\n < boolean[][] array >", fillcolor=white   style=filled  ]
"List[38]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[39]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayInit[40]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[41]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayInit[42]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[43]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[44]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[45]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[46]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[47]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[48]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[49]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[50]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"OrLogicalExpr[51]" [label="OrLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[52]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[53]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[54]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LTExpr[55]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[56]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[57]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"ArrayInit[58]" [label="ArrayInit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[59]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[60]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[61]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[62]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[63]" [label="IntegerLiteral\n < 5 >", fillcolor=white   style=filled  ]
"LTExpr[64]" [label="LTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[65]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[66]" [label="IntegerLiteral\n < 6 >", fillcolor=white   style=filled  ]
"Exit[28]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"Modifiers[19]" -> "List[20]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "Modifiers[19]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "PrimitiveTypeAccess[21]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[22]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[26]" [style=dashed, color=gray]
"Modifiers[31]" -> "List[32]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "Modifiers[31]" [style=dashed, color=gray]
"ArrayTypeAccess[34]" -> "PrimitiveTypeAccess[35]" [style=dashed, color=gray]
"ArrayTypeAccess[33]" -> "ArrayTypeAccess[34]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "ArrayTypeAccess[33]" [style=dashed, color=gray]
"VariableDeclarator[37]" -> "List[38]" [style=dashed, color=gray]
"GTExpr[45]" -> "VarAccess[46]" [style=dashed, color=gray]
"GTExpr[45]" -> "IntegerLiteral[47]" [style=dashed, color=gray]
"AndLogicalExpr[44]" -> "GTExpr[45]" [style=dashed, color=gray]
"LTExpr[48]" -> "VarAccess[49]" [style=dashed, color=gray]
"LTExpr[48]" -> "IntegerLiteral[50]" [style=dashed, color=gray]
"AndLogicalExpr[44]" -> "LTExpr[48]" [style=dashed, color=gray]
"List[43]" -> "AndLogicalExpr[44]" [style=dashed, color=gray]
"GTExpr[52]" -> "VarAccess[53]" [style=dashed, color=gray]
"GTExpr[52]" -> "IntegerLiteral[54]" [style=dashed, color=gray]
"OrLogicalExpr[51]" -> "GTExpr[52]" [style=dashed, color=gray]
"LTExpr[55]" -> "VarAccess[56]" [style=dashed, color=gray]
"LTExpr[55]" -> "IntegerLiteral[57]" [style=dashed, color=gray]
"OrLogicalExpr[51]" -> "LTExpr[55]" [style=dashed, color=gray]
"List[43]" -> "OrLogicalExpr[51]" [style=dashed, color=gray]
"ArrayInit[42]" -> "List[43]" [style=dashed, color=gray]
"List[41]" -> "ArrayInit[42]" [style=dashed, color=gray]
"GTExpr[61]" -> "VarAccess[62]" [style=dashed, color=gray]
"GTExpr[61]" -> "IntegerLiteral[63]" [style=dashed, color=gray]
"AndLogicalExpr[60]" -> "GTExpr[61]" [style=dashed, color=gray]
"LTExpr[64]" -> "VarAccess[65]" [style=dashed, color=gray]
"LTExpr[64]" -> "IntegerLiteral[66]" [style=dashed, color=gray]
"AndLogicalExpr[60]" -> "LTExpr[64]" [style=dashed, color=gray]
"List[59]" -> "AndLogicalExpr[60]" [style=dashed, color=gray]
"ArrayInit[58]" -> "List[59]" [style=dashed, color=gray]
"List[41]" -> "ArrayInit[58]" [style=dashed, color=gray]
"ArrayInit[40]" -> "List[41]" [style=dashed, color=gray]
"Opt[39]" -> "ArrayInit[40]" [style=dashed, color=gray]
"VariableDeclarator[37]" -> "Opt[39]" [style=dashed, color=gray]
"List[36]" -> "VariableDeclarator[37]" [style=dashed, color=gray]
"VarDeclStmt[30]" -> "List[36]" [style=dashed, color=gray]
"List[29]" -> "VarDeclStmt[30]" [style=dashed, color=gray]
"Block[28]" -> "List[29]" [style=dashed, color=gray]
"Opt[27]" -> "Block[28]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[27]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[28]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[28]" -> "VariableDeclarator[37]" [color=red, constraint=false]
"VariableDeclarator[37]" -> "Exit[28]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[37]" -> "GTExpr[61]" [color=red, constraint=false]
"VariableDeclarator[37]" -> "LTExpr[64]" [color=red, constraint=false]
"LTExpr[64]" -> "VariableDeclarator[37]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[64]" -> "IntegerLiteral[66]" [color=red, constraint=false]
"IntegerLiteral[66]" -> "LTExpr[64]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[66]" -> "VarAccess[65]" [color=red, constraint=false]
"VarAccess[65]" -> "IntegerLiteral[66]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[65]" -> "GTExpr[61]" [color=red, constraint=false]
"GTExpr[61]" -> "VarAccess[65]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[61]" -> "VariableDeclarator[37]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[61]" -> "IntegerLiteral[63]" [color=red, constraint=false]
"IntegerLiteral[63]" -> "GTExpr[61]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[63]" -> "VarAccess[62]" [color=red, constraint=false]
"VarAccess[62]" -> "IntegerLiteral[63]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[62]" -> "GTExpr[52]" [color=red, constraint=false]
"VarAccess[62]" -> "LTExpr[55]" [color=red, constraint=false]
"LTExpr[55]" -> "VarAccess[62]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[55]" -> "IntegerLiteral[57]" [color=red, constraint=false]
"IntegerLiteral[57]" -> "LTExpr[55]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[57]" -> "VarAccess[56]" [color=red, constraint=false]
"VarAccess[56]" -> "IntegerLiteral[57]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[56]" -> "GTExpr[52]" [color=red, constraint=false]
"GTExpr[52]" -> "VarAccess[62]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[52]" -> "VarAccess[56]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[52]" -> "IntegerLiteral[54]" [color=red, constraint=false]
"IntegerLiteral[54]" -> "GTExpr[52]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[54]" -> "VarAccess[53]" [color=red, constraint=false]
"VarAccess[53]" -> "IntegerLiteral[54]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[53]" -> "GTExpr[45]" [color=red, constraint=false]
"VarAccess[53]" -> "LTExpr[48]" [color=red, constraint=false]
"LTExpr[48]" -> "VarAccess[53]" [color=blue, constraint=false, xlabel=" "] 
"LTExpr[48]" -> "IntegerLiteral[50]" [color=red, constraint=false]
"IntegerLiteral[50]" -> "LTExpr[48]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[50]" -> "VarAccess[49]" [color=red, constraint=false]
"VarAccess[49]" -> "IntegerLiteral[50]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[49]" -> "GTExpr[45]" [color=red, constraint=false]
"GTExpr[45]" -> "VarAccess[49]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[45]" -> "VarAccess[53]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[45]" -> "IntegerLiteral[47]" [color=red, constraint=false]
"IntegerLiteral[47]" -> "GTExpr[45]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[47]" -> "VarAccess[46]" [color=red, constraint=false]
"VarAccess[46]" -> "IntegerLiteral[47]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[46]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VarAccess[46]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[26]" -> "Opt[27]" -> "Exit[28]" [style=invis]
"ParameterDeclaration[18]" -> "ParameterDeclaration[22]" [style=invis]
"Modifiers[19]" -> "PrimitiveTypeAccess[21]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" [style=invis]
"Modifiers[31]" -> "ArrayTypeAccess[33]" -> "List[36]" [style=invis]
"List[38]" -> "Opt[39]" [style=invis]
"ArrayInit[42]" -> "ArrayInit[58]" [style=invis]
"AndLogicalExpr[44]" -> "OrLogicalExpr[51]" [style=invis]
"GTExpr[45]" -> "LTExpr[48]" [style=invis]
"VarAccess[46]" -> "IntegerLiteral[47]" [style=invis]
"VarAccess[49]" -> "IntegerLiteral[50]" [style=invis]
"GTExpr[52]" -> "LTExpr[55]" [style=invis]
"VarAccess[53]" -> "IntegerLiteral[54]" [style=invis]
"VarAccess[56]" -> "IntegerLiteral[57]" [style=invis]
"GTExpr[61]" -> "LTExpr[64]" [style=invis]
"VarAccess[62]" -> "IntegerLiteral[63]" [style=invis]
"VarAccess[65]" -> "IntegerLiteral[66]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
68", fillcolor=1]
Edges [label="Total Nr. Edges
113", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
21", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
23", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}