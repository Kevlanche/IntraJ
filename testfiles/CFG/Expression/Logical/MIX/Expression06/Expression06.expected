NodeName: Entry[13] Successors: [VarAccess[46]]
NodeName: VarAccess[46] Successors: [IntegerLiteral[47]]
NodeName: IntegerLiteral[47] Successors: [GTExpr[45]]
NodeName: GTExpr[45] Successors: [VarAccess[49], VarAccess[53]]
NodeName: VarAccess[49] Successors: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Successors: [LTExpr[48]]
NodeName: LTExpr[48] Successors: [VarAccess[53]]
NodeName: VarAccess[53] Successors: [IntegerLiteral[54]]
NodeName: IntegerLiteral[54] Successors: [GTExpr[52]]
NodeName: GTExpr[52] Successors: [VarAccess[62], VarAccess[56]]
NodeName: VarAccess[62] Successors: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Successors: [GTExpr[61]]
NodeName: GTExpr[61] Successors: [VarAccess[65], VariableDeclarator[37]]
NodeName: VarAccess[65] Successors: [IntegerLiteral[66]]
NodeName: IntegerLiteral[66] Successors: [LTExpr[64]]
NodeName: LTExpr[64] Successors: [VariableDeclarator[37]]
NodeName: VariableDeclarator[37] Successors: [Exit[28]]
NodeName: Exit[28] Successors: []
NodeName: VarAccess[56] Successors: [IntegerLiteral[57]]
NodeName: IntegerLiteral[57] Successors: [LTExpr[55]]
NodeName: LTExpr[55] Successors: [VarAccess[62]]

NodeName: Exit[28] Predecessor: [VariableDeclarator[37]]
NodeName: VariableDeclarator[37] Predecessor: [GTExpr[61], LTExpr[64]]
NodeName: GTExpr[61] Predecessor: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Predecessor: [VarAccess[62]]
NodeName: VarAccess[62] Predecessor: [GTExpr[52], LTExpr[55]]
NodeName: GTExpr[52] Predecessor: [IntegerLiteral[54]]
NodeName: IntegerLiteral[54] Predecessor: [VarAccess[53]]
NodeName: VarAccess[53] Predecessor: [GTExpr[45], LTExpr[48]]
NodeName: GTExpr[45] Predecessor: [IntegerLiteral[47]]
NodeName: IntegerLiteral[47] Predecessor: [VarAccess[46]]
NodeName: VarAccess[46] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: LTExpr[48] Predecessor: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Predecessor: [VarAccess[49]]
NodeName: VarAccess[49] Predecessor: [GTExpr[45]]
NodeName: LTExpr[55] Predecessor: [IntegerLiteral[57]]
NodeName: IntegerLiteral[57] Predecessor: [VarAccess[56]]
NodeName: VarAccess[56] Predecessor: [GTExpr[52]]
NodeName: LTExpr[64] Predecessor: [IntegerLiteral[66]]
NodeName: IntegerLiteral[66] Predecessor: [VarAccess[65]]
NodeName: VarAccess[65] Predecessor: [GTExpr[61]]
